tests/core/backend_connection/test_authenticated_conn.py

```diff
@@ -168,6 +169,7 @@ async def test_init_with_backend_offline(event_bus, alice):
                 await conn.cmds.ping("foo")
 
 
+@pytest.mark.skipif(OXIDIZED, reason="No error")
```

Comment:
Why is that so ?

this test is about the monitor crashing, which induce the fact authenticated conn is no longer available in such case. Why is this different when using the oxidized system ?

(I already know the answer, but the reader of this code doesn't ^^)